### ISSUE TITLE
Using accessiblity mode a property 'textContent' can be undefined

### ISSUE THREAD

**Issue by marusak** [xtermjs/xterm.js#2082]
Using accessibility mode can sometime end up with trace as:
```
TypeError: Cannot set property 'textContent' of undefined
at AccessibilityManager../node_modules/xterm/lib/AccessibilityManager.js.AccessibilityManager._renderRows (...)
at RenderDebouncer../node_modules/xterm/lib/ui/RenderDebouncer.js.RenderDebouncer._innerRefresh (...)
```

This is new regression seen only in version `3.13.0`, version `3.12.2` was not affected.
In general this is hard to reproduce, it happens in our CI, or I can only reproduce it with very high load on my machine and running tests in loop.

Possible place of happening https://github.com/xtermjs/xterm.js/blob/master/src/AccessibilityManager.ts#L252

I know this is no reproducer, possibly just conditionally check that these elements are present? Seems related to https://github.com/xtermjs/xterm.js/commit/4c3082c924ac or more general to https://github.com/xtermjs/xterm.js/pull/2026
